pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -948,7 +937,7 @@ func (c *Reconciler) createTaskRun(ctx context.Context, taskRunName string, para
 		tr.Annotations[workspace.AnnotationAffinityAssistantName] = getAffinityAssistantName(pipelinePVCWorkspaceName, pr.Name)
 	}
 
-	resources.WrapSteps(&tr.Spec, rpt.PipelineTask, rpt.ResolvedTaskResources.Inputs, rpt.ResolvedTaskResources.Outputs, storageBasePath)
+	resources.WrapSteps(&tr.Spec, rpt.PipelineTask, rpt.ResolvedTaskResources.Inputs, rpt.ResolvedTaskResources.Outputs)
```

Comment:
@JeromeJu are you still having trouble with this bit? I think the git pipelineresource also uses artifact storage; it might be easier to not delete artifact storage in this PR or to delete the git resource and the storage resource in the same PR

Reply:
Thanks Lee, that makes sense, I tried to merge the changes and restore the parts on Git resources to be restored for image type at https://github.com/tektoncd/pipeline/pull/6150. 

And if that looks good I will try to make Image Resources removal as the last PR along with the removal of the generic functions as the removal of both Storage and Git resources has already been a large chunk.

Reply:
Thanks @lbernick , that makes sense, I tried to merge the changes and restore the parts on Git resources to be restored for image type at https://github.com/tektoncd/pipeline/pull/6150. 

And if that looks good I will try to make Image Resources removal as the last PR along with the removal of the generic functions as the removal of both Storage and Git resources has already been a large chunk.